docs/Zendesk-batchsource.md

```diff
@@ -56,7 +49,19 @@ If no value is provided, no upper bound is applied.
 **Satisfaction Ratings Score:** Filter Satisfaction Ratings object to include only records that have a Zendesk score
 equal to the specified score. Only applicable for the Satisfaction Rating object.
 
-### Advanced
+### Connection
+
+**Use Connection:** Whether to use a connection. If a connection is used, you do not need to provide the credentials.
```

Comment:
Usually, Use Connection and Connection appear first in the plugin UI? .md order should match the UI order.
Why are we adding a section called Connection? I haven't seen this in the other plugins...

Reply:
For better visibility we have added the label Connection and order is correct according to that